lib/annotate_rb/model_annotator/model_files_getter.rb

```diff
@@ -9,13 +9,9 @@ class << self
         # of model files from root dir. Otherwise we take all the model files
         # in the model_dir directory.
         def call(options)
-          model_files = []
+          model_files = list_model_files_from_argument(options)
 
-          # Note: This is currently broken as we don't set `is_rake` anywhere.
-          # It's an artifact from the old Annotate gem and how it did control flow.
-          model_files = list_model_files_from_argument(options) if !options[:is_rake]
```

Comment:
`:is_rake` is a legacy option from the old Annotate gem, it's not used in this gem.